app/assets/stylesheets/admin/orders.css.scss

```diff
@@ -87,3 +87,7 @@ div#group_buy_calculation {
 th.actions {
   white-space: nowrap;
 }
+
+table.index td.actions {
+  text-align: left;
+}
```

Comment:
I'm not a CSS expert myself but Code Climate complaints about the use of the `table` and `td` element selectors.